### PR TITLE
Removes one line ifs

### PR DESCRIPTION
**Pull request by ComicIronic**

### CODE REVIEW

**Comment by Exxion** [comment]
Oh Jesus fuck.

**Comment by clusterfack** [comment]
The Absolut™ madman

**Comment by Exxion** [comment]
That second tag almost applies.

**Comment by Venseer** [comment]
I'm surprised that there was only 917 file changes.

**Comment by ComicIronic** [comment]
Every time someone tries to defend themselves with a historical example, I make a regex PR.

**Comment by Exxion** [comment]
Did you take commented-out code into account?

**Comment by ComicIronic** [comment]
I did not. Should have anchored the regex to linestart.

**Comment by unid15** [comment]
[meme] tag when

**Comment by 9600bauds** [comment]
I wonder when are we going to channel any of this initiative into something that actually matters, like "loc =". We're in a long streak of useless shit only.

**Comment by ComicIronic** [comment]
Make a list of things that need mass changes. loc is a relatively easy one to do.

**Comment by clusterfack** [comment]
Well the new rule about standards is 'only standardized things can be standards', so yeah I support this

**Comment by ghost** [comment]
> Well the new rule about standards is 'only standardized things can be standards', so yeah I support this

why are we still removing trailing returns then? :^)

File: code/WorkInProgress/SkyMarshal/officer_stuff.dm [hunk 1]
```diff
@@ -3,13 +3,13 @@
 
 /obj/item/clothing/head/beret/centcom/officer
 	name = "officers beret"
-	desc = "A black beret adorned with the shield�a silver kite shield with an engraved sword�of the Nanotrasen security forces, announcing to the world that the wearer is a defender of Nanotrasen."
+	desc = "A black beret adorned with the shield�a silver kite shield with an engraved sword�of the Nanotrasen security forces, announcing to the world that the wearer is a defender of Nanotrasen."
```
Comment (Kurfursten) [inline code/WorkInProgress/SkyMarshal/officer_stuff.dm]: Non atomic

**Comment by StarSmasher44** [comment]
So just wondering is there any actual benefit to turning one-liners into 2-liners beside having more lines in total? Not trying to bash anyone's work obviously, just wondering.

**Comment by Exxion** [comment]
More readable, very very slightly less likely to conflict

**Comment by 9600bauds** [comment]
Less likely to conflict how?

**Comment by Exxion** [comment]
If one person changes the check in a one-line if and someone else makes it multiple lines to add something else to the block
I did say "very very slightly"

File: code/game/machinery/bots/mulebot.dm [hunk 2]
```diff
@@ -530,10 +531,12 @@ var/global/mulebot_count = 0
...
 				process_bot()
 
-	if(refresh) updateDialog()
+	if(refresh)
+		updateDialog()
 
 /obj/machinery/bot/mulebot/proc/process_bot()
-//	to_chat(if(mode) world, "Mode: [mode]")
+//	to_chat(if(mode)
+	world, "Mode: [mode]")
```
Comment (unid15) [inline code/game/machinery/bots/mulebot.dm]: non-atomic

File: code/game/machinery/requests_console.dm [hunk 3]
```diff
@@ -374,7 +380,8 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 	//Handle silencing the console
 	switch( href_list["setSilent"] )
 		if(null)	//skip
-		if("1")	silent = 1
+		if("1")
+			silent = 1
 		else	silent = 0
```
Comment (unid15) [inline code/game/machinery/requests_console.dm]: no one-line elses

File: code/game/machinery/shieldgen.dm [hunk 4]
```diff
@@ -57,15 +59,17 @@
 		return
 
 	opacity = 1
-	spawn(20) if(src) opacity = 0
+	spawn(20) if(src)
+ 	opacity = 0
```
Comment (unid15) [inline code/game/machinery/shieldgen.dm]: non atomic

**Comment by ComicIronic** [comment]
DM is, by most definitions, and despite Tom's best efforts, a whitespace language. It should be using whitespace formatting as a standard for control flow.

File: code/WorkInProgress/SkyMarshal/officer_stuff.dm (hunk 1 above)
Comment (kol1th) [inline code/WorkInProgress/SkyMarshal/officer_stuff.dm]: C O M I C A T O M I C

**Comment by ghost** [comment]
> 

But surely if you were just removing one line elses ifs and such, the additions would be double the removed lines?

**Comment by ComicIronic** [comment]
There were several cases where removing the one line if led to further formatting changes.

This is totally valid DM

```
var/a = 1, b = 2, c = 3
if(true) { a=3; b=7; c=15}
```

Which I had to restructure as 

```
var/a = 1, b = 2, c = 3
if(true)
    a=3
    b=7
    c=15
```

**Comment by clusterfack** [comment]
A standard aint a standard lest its standard

**Comment by ghost** [comment]
@clusterfack that's not even true though because of trailing returns still existing

**Comment by clusterfack** [comment]
I sense another pr on the horizon

**Comment by ComicIronic** [comment]
Removing trailing returns would be extremely difficult without a true DM parser. So, I'm not doing it.

**Comment by PJB3005** [comment]
Yep, in most cases it can be done but in a lot of others it's basically impossible to do automatically.

**Comment by clusterfack** [comment]
Really? I didn't think that would be a very complex regex, considering the format is always a single line with just a tab and return on it. Nothing else should be picked up by that either.

**Comment by unid15** [comment]
#10738 reopened when

**Comment by ComicIronic** [comment]
#10738 would DEFINITELY require a parser to check where src was actually necessary.

**Comment by ghost** [comment]
so trailing returns are not a standard, got it lads

**Comment by clusterfack** [comment]
Don't worry thats coming up next celt

**Comment by ghost** [comment]
@clusterfack no, but it isn't though

**Comment by clusterfack** [comment]
@DrCelt I'm sure we'll have that sorted by the time your month long absence ends this time

**Comment by ghost** [comment]
Two week long.

**Pull request merged**